site/docs/how-to/aws.mdx

```diff
@@ -49,3 +49,36 @@ SELECT
 FROM
   iasql_install ('aws_iam', 'aws_cloudwatch', 'aws_ecr', 'aws_ecs_fargate', 'aws_elb', 'aws_security_group', 'aws_vpc');
 ```
+
+## Restricting AWS credentials to specific regions
+
+Sometimes, you may want or need to restrict access to a specific region. To do this, you will need to make sure that the credentials for the IAM role given to IaSQL have a policy that restricts access only to AWS resources in that region. This can be done by creating a custom policy and attaching it to the relevant IAM role. The following [guide](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_create.html) will walk you through creating a new policy. Your policy will need to look something like this:
+
+```json
```

Comment:
One comment on this policy: It will only work as explained here if the IAM user in question doesn't have any policy giving access to AWS regions, yet. If that IAM user already has access to all regions, this policy will do nothing.

Perhaps there should be a second statement with an `"Effect": "Deny"` and the condition the mutated to `StringNotEquals`? (Which I just confirmed [is a real conditional](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_condition_operators.html) )